src/utils/getNewTodoId.ts

```diff
@@ -0,0 +1,7 @@
+import { Todo } from '../types/Todo';
+
+export function getNewTodoId(todos: Todo[]) {
+  const maxId = Math.max(...todos.map(todo => todo.id));
```

Comment:
[CODE QUALITY] - When using `Math.max` with an empty array, it will return `-Infinity`, which will lead to an incorrect ID when the `todos` array is empty. Consider handling the case when `todos` is empty to ensure that a valid ID is always returned. For instance, you could default `maxId` to `0` when the array is empty.